client/src/components/TaskStatusChart/TaskStatusChart.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import styles from "./TaskStatusChart.module.css";
+import { PieChart, Pie, Cell, Legend } from "recharts";
+import { PropTypes } from "prop-types";
+
+const RADIAN = Math.PI / 180;
+const renderCustomizedLabel = ({
+  cx,
```

Comment:
Just wondering, what does cy and cx represent?

Reply:
Nevermind, I found where the code was referenced from
https://recharts.org/en-US/examples/PieChartWithCustomizedLabel (Keeping this here for future reference)